lib/time.js

```diff
@@ -0,0 +1,17 @@
+const fetch = require("node-fetch");
+const CONSTANTS = require("../utils/constants");
+
+async function getTime(){
+    const url = `${CONSTANTS.baseUrl}/time`
+    const response = await fetch(url, {
+        method: 'GET',
+        headers: {
+        "Content-Type": "application/json",
+        },
+    });
+
+    const unixResponse = await response.json();
+    return unixResponse.time;
```

Comment:
try to have a response object for all API calls that exposes all results. Some might be interested to know the timezone as well.